cmd/fitgen/internal/profile/codegen.go

```diff
@@ -209,6 +230,7 @@ func (g *codeGenerator) genConstructor(msg *Msg) {
 	for _, f := range msg.Fields {
 		g.p(f.CCName, ": ", f.FType.GoInvalidValue(), ",")
 	}
+	g.p("DeveloperFields: map[string]DeveloperField{},")
```

Comment:
it might be nice to leave this as `nil` and initialize it on the first dev field

Reply:
I did this since the pattern is to have a fully initialized struct returned from these constructor funcs.